1. undefined reference to `free'

2. Cherry pick trunk@13760 to remove boost::system
   per https://marc.info/?l=openbsd-ports&m=177038373917003&w=2

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -15,7 +15,6 @@ AC_MSG_RESULT([${enable_flags_setting}])
 m4_include([m4/acinclude.m4])
 m4_include([m4/ax_cxx_compile_stdcxx.m4])
 m4_include([m4/ax_boost_base.m4])
-m4_include([m4/ax_boost_system.m4])
 
 CODEBLOCKS_CHECK_DEBUG
 
@@ -43,8 +42,6 @@ AC_PROG_MAKE_SET
 AC_PROG_AWK
 
 AX_CXX_COMPILE_STDCXX(11, noext)
-
-CODEBLOCKS_CHECK_LINK_FLAG([-Wl,--no-undefined],[LDFLAGS="-Wl,--no-undefined $LDFLAGS"])
 
 CODEBLOCKS_SETUP_FOR_TARGET
 
